### ISSUE TITLE
[Adaptive Card iOS - InputNumber.json]: Neither Voice over is announcing the required information nor visual asterisks mark is present for the input number edit field. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#190]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'InputNumber.Jsons' and activate it.
7. The Card will open.
8. Perform right swipe and navigate to edit field,
9. Made the field blank and perform the right swipe and navigate to the Submit button and double tap on it and observe the issue.

**Actual Result:**
While navigating on the 'Input.Edit' field, Voice over neither announcing the required information nor there is visually asterisk mark is present for it.

**Expected Result:**
There should be a visual asterisk mark should be present for the required field or when the focus lands on it then screen reader should announce the required information for the required field.

**User Impact:**
It will impact the screen reader user as they will not able to know whether the field is required or not due to this there might be a chance to skip the required field.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#labels-or-instructions

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/612c10f9-b5e3-45a4-b2c1-fade93fbc09d

### Card JSON

```JSON
NA
```
